### ISSUE TITLE
Esc doesn't work when navigating Mixer submenus

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#32343]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Navigate Mixer>Sound>MS Basic>..
2. Hit Esc

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/3b44951b-73f0-4ef3-8d7b-6627be1b8e1c

### In which versions of MuseScore Studio is this issue present?

4.7 beta builds (4.7.0-260222043)

### Regression

Yes, this works in the latest released version and is broken in the nightly builds

### Operating system

Windows 10

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
